controllers/install/lookout_controller.go

```diff
@@ -49,7 +54,8 @@ type LookoutReconciler struct {
 
 //+kubebuilder:rbac:groups=install.armadaproject.io,resources=lookouts,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=install.armadaproject.io,resources=lookouts/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups="batch",resources=jobs,verbs=get;list;watch;create;delete;deletecollection;patch;update
+//+kubebuilder:rbac:groups="batch",resources=jobs;cronjobs,verbs=get;list;watch;create;delete;deletecollection;patch;update
+//+kubebuilder:rbac:groups="";resources=prometheusrules,verbs=get;list;create;update;patch;delete
```

Comment:
I don't think this would work.  Groups references the API group and prometheusrules is not part of the core.  

"" means core Kubernetes.  

PrometheusRules is part of the operator so we will have to add that in.